changes/ee/fix-15067.en.md

```diff
@@ -0,0 +1,3 @@
+Fixed a couple of issues with Cluster Linking Route Replication.
+* Replication process may enter reconnect loop after Cluster Link becomes correctly configured from both sides.
```

Comment:
correctly or incorrectly ?

Reply:
"Correctly". But I'll elaborate this point.